app.py

```
@@ -1,103 +1,404 @@
+                        {"role": "user", "content": prompt}
+                    ],
+                    max_tokens=150,
+                    temperature=0.7
+                )
+
+                return response.choices[0].message.content.strip()
+            else:
+                # Fallback if OpenAI is not configured
+                return f"Just shipped: {commit_data.commit_message} 🚀\n\nWorking on {commit_data.repository_name} - {files_text}\n\nWhat's everyone else building today?"
+
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)